bundles/org.openhab.binding.sony/src/main/java/org/openhab/binding/sony/internal/scalarweb/protocols/ScalarWebAvContentProtocol.java

```diff
@@ -0,0 +1,3223 @@
+                        "scalarwebavcontrolplsource", prefix + "Source", null));
+
+                descriptors.add(createDescriptor(createChannel(PL_STARTDATETIME, id, uri), "String",
+                        "scalarwebavcontrolplstartdatetime", prefix + "Start Date/Time", null));
+
+                descriptors.add(createDescriptor(createChannel(PL_TITLE, id, uri), "String",
+                        "scalarwebavcontrolpltitle", prefix + "Title", null));
+
+                descriptors.add(createDescriptor(createChannel(PL_TRIPLETSTR, id, uri), "String",
+                        "scalarwebavcontrolpltripletstr", prefix + "Triplet", null));
+
+                descriptors.add(createDescriptor(createChannel(PL_URI, id, uri), "String", "scalarwebavcontrolpluri",
+                        prefix + "URI", null));
+
+            }
+
+            if (VersionUtilities.equals(version, ScalarWebMethod.V1_2)) {
+                descriptors.add(createDescriptor(createChannel(PL_ALBUMNAME, id, uri), "String",
+                        "scalarwebavcontrolplalbumname", prefix + "Album Name", null));
+                descriptors.add(createDescriptor(createChannel(PL_APPLICATIONNAME, id, uri), "String",
+                        "scalarwebavcontrolplapplicationname", prefix + "Application Name", null));
+                descriptors.add(createDescriptor(createChannel(PL_ARTIST, id, uri), "String",
+                        "scalarwebavcontrolplartist", prefix + "Artist", null));
+                descriptors.add(createDescriptor(createChannel(PL_AUDIOCHANNEL, id, uri), "String",
+                        "scalarwebavcontrolplaudiochannel", prefix + "Audio Channel", null));
+                descriptors.add(createDescriptor(createChannel(PL_AUDIOCODEC, id, uri), "String",
+                        "scalarwebavcontrolplaudiocodec", prefix + "Audio Codec", null));
+                descriptors.add(createDescriptor(createChannel(PL_AUDIOFREQUENCY, id, uri), "String",
+                        "scalarwebavcontrolplaudiofrequency", prefix + "Audio Frequency", null));
+                descriptors.add(createDescriptor(createChannel(PL_BROADCASTFREQ, id, uri), "Number",
+                        "scalarwebavcontrolplbroadcastfreq", prefix + "Broadcast Frequency", null));
+                descriptors.add(createDescriptor(createChannel(PL_BROADCASTFREQBAND, id, uri), "String",
+                        "scalarwebavcontrolplbroadcastfreqband", prefix + "Broadcast Frequency Band", null));
+                descriptors.add(createDescriptor(createChannel(PL_CHANNELNAME, id, uri), "String",
+                        "scalarwebavcontrolplchannelname", prefix + "Channel Name", null));
+                descriptors.add(createDescriptor(createChannel(PL_CHAPTERCOUNT, id, uri), "Number",
+                        "scalarwebavcontrolplchaptercount", prefix + "Chapter Count", null));
+                descriptors.add
```

Comment:
So you are always making 1 additional request than would seem necessary?

Reply:
Not sure I'm following what you are saying here

Reply:
my comment is on line 2901. I have no idea why github is showing it as line 997 here...

Reply:
yikes - I wrong that so long ago I never relooked at it and you are absolutely correct (actually - that was pretty sucky code - don't know what I was attempting there).  Fixed